api/v1/types/model_types.go

```diff
@@ -50,10 +50,12 @@ type StatsResponse struct {
 	Stats  map[string]ExternalStatBundle `json:"stats"`
 }
 
-// An ExternalEntityListEntry represents the latest CPU and Memory usage of a model entity.
+// An ExternalEntityListEntry represents the latest CPU, Memory & network usage of a model entity.
 // A model entity can be a Pod, a Container, a Namespace or a Node.
 type ExternalEntityListEntry struct {
 	Name     string `json:"name"`
 	CPUUsage uint64 `json:"cpuUsage"`
 	MemUsage uint64 `json:"memUsage"`
+	RxBytes  uint64 `json:"rx_bytes"`
```

Comment:
Point me to the requirements :) Don't we make it up as we go along? ;)

Honestly, if it's too expensive to store a couple of extra values per pod then I don't mind dropping it from there & just leave it as passed to sinks.